### PR TITLE
Ehab DB Week1

### PR DESCRIPTION
**Pull request by ehabwasel**

### CODE REVIEW

File: Week1/homework/ex1_create_table.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+connection.query(("USE meetup"), (error, results, fields) => {
+  if (error) throw error;
+});
+
+
+const create_Invitee = "create table Invitee (invitee_no int, invitee_name varchar(50),  invited_by varchar(50))";
+
+const create_Room  = "create table Room (room_no int, room_name varchar(50), floor_number int)";
+
+const create_Meeting  = "create table Meeting (meeting_no int, meeting_title varchar(50),  starting_time date , ending_time date)";
```
[the remark this answers is not in the document]
Comment (Breus) [reply Week1/homework/ex1_create_table.js]: Hint: check via MySQL workbench or terminal what the actual inserted values for starting_time and ending_time are ;-)

File: Week1/homework/ex2.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+});
+
+
+connection.connect(err => {
+  if (err) throw err;
+  console.log(" DB world IS connected..");
+});
+
+
+var select_query = [
```
Comment (Breus) [inline Week1/homework/ex2.js:20]: Any reason you used var for this array instead of const?

**Review by Breus — CHANGES_REQUESTED** [review]
Looks good in general, but besides some small code improvements there is one error in your homework in the first exercise (something with the starting and ending times).

File: Week1/homework/ex1_create_table.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+const  connection = mysql.createConnection({
+  host     : 'localhost',
+  user     : 'hyfuser',
+  password : 'hyfpassword',
+  port : 3306
+});
+
+connection.connect(err => {
+  if (err) throw err;
+  console.log(" DB meetup IS connected..");
```
[the remark this answers is not in the document]
Comment (ehabwasel) [reply Week1/homework/ex1_create_table.js]: Ooh yes. Thanks i changed it when i setup and check my sql and i didn't change it. Thanks i changed it naw and i made new commit for your new advices. Thanks 😊

File: Week1/homework/ex1_create_table.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+tables(create_Invitee);
+tables(create_Room);
+tables(create_Meeting);
+
+const array_invitee = ["INSERT INTO Invitee VALUES (1, 'Dan', 'fade')","INSERT INTO Invitee VALUES (2, 'sarah', 'nizami')","INSERT INTO Invitee VALUES (3, 'marwa', 'khalid')","INSERT INTO Invitee VALUES (4, 'atabek', 'ahmed')","INSERT INTO Invitee VALUES (5,'ehab','ali')"];
+
+ const array_room = ["INSERT INTO Room VALUES (1, 'Dan', 10)","INSERT INTO Room VALUES (2, 'manal', 20)","INSERT INTO Room VALUES (3, 'yasmin', 30)","INSERT INTO Room VALUES (4, 'ellen', 40)","INSERT INTO Room VALUES (5, 'asmaa', 40)"];
+
+ const array_meeting = ["INSERT INTO Meeting VALUES (1, 'html', '2020-01-01 10:10:10','2020-01-01 10:10:10')","INSERT INTO Meeting VALUES (2, 'css', '2020-01-01 10:10:10','2020-01-01 10:10:10')","INSERT INTO Meeting VALUES (3, 'js1', '2020-01-01 10:10:10','2020-01-01 10:10:10')","INSERT INTO Meeting VALUES (4, 'JS2', '2020-01-01 10:10:10','2020-01-01 10:10:10')","INSERT INTO Meeting VALUES (5, 'js3', '2020-01-01 10:10:10','2020-01-01 10:10:10')"];
```
[the remark this answers is not in the document]
Comment (ehabwasel) [reply Week1/homework/ex1_create_table.js]: Thanks. I did in the new commit 👍

File: Week1/homework/ex1_create_table.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+});
+
+
+const create_Invitee = "create table Invitee (invitee_no int, invitee_name varchar(50),  invited_by varchar(50))";
+
+const create_Room  = "create table Room (room_no int, room_name varchar(50), floor_number int)";
+
+const create_Meeting  = "create table Meeting (meeting_no int, meeting_title varchar(50),  starting_time date , ending_time date)";
+
+function tables(create_query){
```
[the remark this answers is not in the document]
Comment (ehabwasel) [reply Week1/homework/ex1_create_table.js]: Yes indeed i changed its name to create_tables now 😊

File: Week1/homework/ex1_create_table.js [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+const array_invitee = ["INSERT INTO Invitee VALUES (1, 'Dan', 'fade')","INSERT INTO Invitee VALUES (2, 'sarah', 'nizami')","INSERT INTO Invitee VALUES (3, 'marwa', 'khalid')","INSERT INTO Invitee VALUES (4, 'atabek', 'ahmed')","INSERT INTO Invitee VALUES (5,'ehab','ali')"];
+
+ const array_room = ["INSERT INTO Room VALUES (1, 'Dan', 10)","INSERT INTO Room VALUES (2, 'manal', 20)","INSERT INTO Room VALUES (3, 'yasmin', 30)","INSERT INTO Room VALUES (4, 'ellen', 40)","INSERT INTO Room VALUES (5, 'asmaa', 40)"];
+
+ const array_meeting = ["INSERT INTO Meeting VALUES (1, 'html', '2020-01-01 10:10:10','2020-01-01 10:10:10')","INSERT INTO Meeting VALUES (2, 'css', '2020-01-01 10:10:10','2020-01-01 10:10:10')","INSERT INTO Meeting VALUES (3, 'js1', '2020-01-01 10:10:10','2020-01-01 10:10:10')","INSERT INTO Meeting VALUES (4, 'JS2', '2020-01-01 10:10:10','2020-01-01 10:10:10')","INSERT INTO Meeting VALUES (5, 'js3', '2020-01-01 10:10:10','2020-01-01 10:10:10')"];
+
+function insert_data (array){
+  array.forEach(element => {
+    connection.query(element, function (error, results, fields) {
+      if (error) {
```
[the remark this answers is not in the document]
Comment (ehabwasel) [reply Week1/homework/ex1_create_table.js:83]: Yes of course i will use the same style with curly brackets. I updated it with new commit 😉

File: Week1/homework/ex2.js (hunk 2 above)
Comment (ehabwasel) [reply Week1/homework/ex2.js]: This really mistake in new commit i used const to avoid variable pollution

**Pull request closed without merging**